packages/codemods/src/codeMods/mods/officeToFluentImport/officeToFluentImport.mod.ts

```diff
@@ -3,8 +3,37 @@ import { CodeMod, CodeModResult } from '../../types';
 import { getImportsByPath, repathImport } from '../../utilities/index';
 import { Ok, Err } from '../../../helpers/result';
 
-const searchString = /^office\-ui\-fabric\-react/;
-const newString = '@fluentui/react';
+const uiFabricRename = /^(@uifabric|@fluentui)\//;
+
+// Packages which only need uifabric to be replaced
+// [ /azure-themes/, "" ],
+// [ /example\-data/, "" ],
+// [ /jest\-serializer-merge-styles/, "" ],
+// [ /merge\-styles/, "" ],
+// [ /monaco\-editor/, "" ],
+// [ /react\-cards/, "" ],
+// [ /react\-hooks/, "" ],
+// [ /set\-version/, "" ],
+// [ /test\-utilities/, "" ],
+// [ /theme\-samples/, "" ],
+// [ /utilities/, "" ],
+
+const uiFabricMap: [RegExp, string][] = [
```

Comment:
Just from reading this I'm having a bit of a hard time understanding exactly what regexps result, and whether they'll be specific enough to avoid accidentally pulling in things that aren't meant to match. For that reason it might be better to just define the array of possible options and replacements manually.

Reply:
Thanks for the updates! One nit is that I think we only need to be looking for `@uifabric` scopes on the renamed packages, since I don't believe any of the `@fluentui` packages were renamed. (Including `@fluentui` shouldn't be harmful, it just wouldn't do anything.)